apps/sharedtest/test/unit/l10n/lib/context/langnego_test.js

```diff
@@ -44,3 +44,20 @@ describe('Language negotiation with arguments', function() {
     assert.strictEqual(ctx.supportedLocales[1], 'en-US');
   });
 });
+
+describe('Language negotiation with arguments and unsupported', function() {
+  var ctx;
+
+  beforeEach(function(done) {
+    ctx = new Context();
+    ctx.once(done);
+    ctx.registerLocales('en-US', ['pl']);
+    ctx.requestLocales('pl-PT', 'pl');
```

Comment:
Can you use something else than `pl-PT` here? Maybe `de` or `fr` which I think are often used in other l10n.js tests.